src/index.js

```diff
@@ -178,6 +176,7 @@ class Dropzone extends React.Component {
   }
 
   open() {
+    this.setState({ isFileDialogActive: true });
```

Comment:
@okonet @akkie 

Thank you for your feedback.

```
open() {
    const { fileInputEl } = this;
    this.setState({ isFileDialogActive: true }, () => {
      fileInputEl.value = null;
      fileInputEl.click();
    });
  }
```

This causes infinite loop since the `fileInputEl.click()` make the open method to called again.

Shall I change the `isFileDialogActive` to be class variable rather than a state. I can change the usage from `this.state.isFileDialogActive` to `this.isFileDialogActive` everywhere.

Need suggestions?